libethcore/EVMSchedule.h

```diff
@@ -35,6 +35,7 @@ struct EVMSchedule
     bool eip150Mode = false;
     bool eip158Mode = false;
     bool eip1283Mode = false;
+    bool eip1380Mode = false;
```

Comment:
I would rather simplify this by removing this flag, defining `callSelfGas` as equal to `callGas` in all schedules before Berlin, then setting it to 40 in Berlin only.

Then EVM will need to check only the dest address and no additional flags.

Make sure to change it together with `callGas` in `EIP150Schedule`.

Reply:
@gumb0 Ah good idea!